samples/CuriePME.js

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2017, Intel Corporation.
+
+// Test code to use the Curie Pattern Matching Engine within the Intel Curie Compute
+// on the Arduino 101/tinyTile, which uses the PME API to train and classify data
+console.log("Curie Pattern Matching Engine test...");
+
+// import pme module
+var pme = require("pme");
+
+var training1 = [ 10, 10, 10, 10 ];
```

Comment:
This repeating 10 thing doesn't make any sense to me. I think you get the same result if you only list it once? Is there a point to this?

We need some better examples soon. :)

If you understand what's going on in here you should add copious comments to describe it to the bewildered user... like me! If not, I guess we live with it for now until we do understand.